src/components/SingleStoryCard/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Row, Col, Image } from "react-bootstrap";
+import { BrowserRouter as NavLink } from "react-router-dom";
+import "./index.css";
+
+const SingleStoryCard = (props) => {
```

Comment:
If you know that you need some specific props I think it is better to destructor.

Reply:
@khaldoon9kh yes you c an destruct the props to make it easier for other developers to know what props are getting passed